Session created
Traceback (most recent call last):
  File "d:\OneDrive - LTTS\Desktop\Shavali\main.py", line 66, in <module>
    main()
  File "d:\OneDrive - LTTS\Desktop\Shavali\main.py", line 57, in main
    subscribe_to_values(client.uaclient)
  File "d:\OneDrive - LTTS\Desktop\Shavali\main.py", line 19, in subscribe_to_values
    subscription = client.create_subscription(500, ua.SubscriptionDiagnosticsDataType())
  File "C:\Users\40020507\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\opcua\client\ua_client.py", line 439, in create_subscription
Client disconnected
    self._uasocket.send_request(request, mycallbak)
  File "C:\Users\40020507\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\opcua\client\ua_client.py", line 81, in send_request
    future = self._send_request(request, callback, timeout, message_type)
  File "C:\Users\40020507\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\opcua\client\ua_client.py", line 55, in _send_request
    binreq = struct_to_binary(request)
  File "C:\Users\40020507\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\opcua\ua\ua_binary.py", line 261, in struct_to_binary
    packet.append(to_binary(uatype, val))
  File "C:\Users\40020507\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\opcua\ua\ua_binary.py", line 286, in to_binary
    raise UaError("No known way to pack {} of type {} to ua binary".format(val, uatype))
opcua.ua.uaerrors._base.UaError: No known way to pack 500 of type CreateSubscriptionParameters to ua binary
